lib/commands/screenshots.js

```diff
@@ -9,6 +9,27 @@ export default {
    * @returns {Promise<string>}
    */
   async getScreenshot() {
+    if (this.isWebContext()) {
+      const webScreenshotMode = (await this.settings.getSettings()).webScreenshotMode;
+      switch (_.toLower(webScreenshotMode)) {
+        case 'page':
+        case 'viewport':
+          return await this.remote.captureScreenshot({
+            coordinateSystem: _.capitalize(webScreenshotMode),
```

Comment:
Would be helpful to leave the original available values here or in the remote debugger for a future reference?
https://github.com/WebKit/WebKit/blob/main/Source/JavaScriptCore/inspector/protocol/Page.json#L56

Reply:
it is documented in the remote debugger code (also typed properly)